ui-manchette-with-spacetimechart/src/hooks/useManchetteWithSpaceTimeChart.ts

```diff
@@ -54,12 +62,6 @@ const useManchettesWithSpaceTimeChart = (
 
   const paths = usePaths(projectPathTrainResult, selectedTrain);
 
-  // Memoize timeWindow to avoid recalculation on each render
-  const timeWindow = useMemo(
-    () => computeTimeWindow(projectPathTrainResult),
```

Comment:
Hm. Removing this no longer makes it so all trains are visible with the default zoom, I think? Is this intentional? (cc @clarani who implemented this feature IIRC)

Reply:
You’re right, They asked for a default value of 7500 ms/px, which means we only have 2h-2h30 visible range by default and when we reset the zoom.

Reply:
seems like starting at offset zero is fine all the time, it’s the start of the earliest train.

Reply:
@thibautsailly confirmed it's not desirable to fit the viewport to the selected train.